### DISCUSSION TITLE
Change kernver?

### DISCUSSION THREAD

**Discussion by doofenator** [3kh0/ext-remover#1092] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
Doofenator here, is it possible to change the kernver to allow cryptosmite?
I found [this thing called kvs,](https://brandon421-ops.github.io/KVS/) but the file hosting is down and I'm not sure if it's a prank or not. If possible, I'm aiming for version 0, but version 2 or 1 is fine

**Comment by S-PScripts** [comment]
@Brandon421-ops

**Comment by FunnyCorgi** [comment]
yeah does anyone know of a rehost for that file hosting?

**Comment by S-PScripts** [comment]
kvs back up again
